src/profile.c

```diff
@@ -239,6 +239,49 @@ static void CheckLeaveFunctionsAfterLongjmp(void)
     }
 }
 
+static Obj JsonEscapeString(Obj param)
+{
+    Int lenString = LEN_LIST(param);
+
+    // Massively over-long string
+    Obj     copy = NEW_STRING(lenString * 2 + 2);
```

Comment:
Why add 2, it seems to serve no point?
```suggestion
    Obj     copy = NEW_STRING(lenString * 2);
```